src/client/Client.cc

```diff
@@ -9433,7 +9433,7 @@ int64_t Client::_write(Fh *f, int64_t offset, uint64_t size, const char *buf,
      * change out from under us.
      */
     if (f->flags & O_APPEND) {
-      int r = _lseek(f, 0, SEEK_END);
+      uint64_t r = _lseek(f, 0, SEEK_END);
       if (r < 0) {
```

Comment:
changing to `uint64_t` means this above comparison is never true. It would be better to automatically use the type returned by `_lseek`, i.e.:

`auto r = _lseek(...);`

Reply:
I also think this is the best way to use auto

Reply:
I have changed it, please review.